app-connector-client/server/resources/api.yaml

```diff
@@ -293,6 +396,37 @@ paths:
 
 components:
   schemas:
+    Status:
+      type: "object"
+      properties:
+        success_count:
+          type: "integer"
+          example: 0
+          description: "The number of apis successfully registered"
+        failed_count:
+          type: "integer"
+          example: 0
+          description: "The number of apis failed to register"
+        apis_success:
+          type: "array"
+          items:
+            $ref: "#/components/schemas/Service"
+        apis_failed:
+          type: "array"
+          items:
+            $ref: "#/components/schemas/Service"
+        apis:
+          type: "array"
+          items:
+            $ref: "#/components/schemas/Service"
+        progress:

```

Comment:
why not just a single attribute "status" being an enum with well defined static values?
Sounds very UI driven what you did here, please remember that it will be used by tests as well.

Reply:
used only a single boolean called inprogress